{
	"alias": "FEDORA-2024-c4a71dab58",
	"autotime": true,
	"bugs": [
		{
			"bug_id": 2275991,
			"security": true,
			"title": "CVE-2024-27306 python-aiohttp: aiohttp: XSS on index pages for static file handling [fedora-all]",
			"bugzilla": {
				"bug_id": "2275991",
				"creation_ts": "2024-04-18 20:35:30 +0000",
				"short_desc": "CVE-2024-27306 python-aiohttp: aiohttp: XSS on index pages for static file handling [fedora-all]",
				"delta_ts": "2024-05-02 01:57:00 +0000",
				"bug_status": "CLOSED",
				"resolution": "ERRATA",
				"keywords": "Security, SecurityTracking",
				"priority": "medium",
				"bug_severity": "medium",
				"blocked": [
					{
						"bug_id": "2275989",
						"alias": "CVE-2024-27306",
						"creation_ts": "2024-04-18 20:28:45 +0000",
						"short_desc": "CVE-2024-27306 aiohttp: XSS on index pages for static file handling",
						"delta_ts": "2024-08-07 04:43:55 +0000",
						"bug_status": "NEW",
						"keywords": "Security",
						"priority": "medium",
						"bug_severity": "medium",
						"depends_on": [
							"2275990",
							"2275991",
							"2275992"
						],
						"blocked": [
							{
								"bug_id": "2275987",
								"error": "NotPermitted"
							}
						],
						"external_bugs": {
							"text": "RHSA-2024:3781",
							"name": "Red Hat Product Errata"
						},
						"long_desc": [
							{
								"isprivate": "0",
								"commentid": "17978083",
								"comment_count": "0",
								"who": {
									"text": "rgatica",
									"name": "Robb Gatica"
								},
								"bug_when": "2024-04-18 20:28:45 +0000",
								"thetext": "aiohttp is an asynchronous HTTP client/server framework for asyncio and Python. A XSS vulnerability exists on index pages for static file handling. This vulnerability is fixed in 3.9.4. We have always recommended using a reverse proxy server (e.g. nginx) for serving static files. Users following the recommendation are unaffected. Other users can disable `show_index` if unable to upgrade.\n\nhttps://github.com/aio-libs/aiohttp/commit/28335525d1eac015a7e7584137678cbb6ff19397\nhttps://github.com/aio-libs/aiohttp/pull/8319\nhttps://github.com/aio-libs/aiohttp/security/advisories/GHSA-7gpw-8wmc-pm8g"
							},
							{
								"isprivate": "0",
								"commentid": "17978095",
								"comment_count": "1",
								"who": {
									"text": "rgatica",
									"name": "Robb Gatica"
								},
								"bug_when": "2024-04-18 20:37:27 +0000",
								"thetext": "Created python-aiohttp tracking bugs for this issue:\n\nAffects: epel-all [bug 2275990]\nAffects: fedora-all [bug 2275991]"
							},
							{
								"isprivate": "0",
								"commentid": "18060147",
								"comment_count": "4",
								"who": {
									"text": "errata-xmlrpc",
									"name": "errata-xmlrpc"
								},
								"bug_when": "2024-06-10 18:37:46 +0000",
								"thetext": "This issue has been addressed in the following products:\n\n  Red Hat Ansible Automation Platform 2.4 for RHEL 9\n  Red Hat Ansible Automation Platform 2.4 for RHEL 8\n\nVia RHSA-2024:3781 https://access.redhat.com/errata/RHSA-2024:3781"
							}
						]
					}
				],
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "17978091",
						"comment_count": "0",
						"who": {
							"text": "rgatica",
							"name": "Robb Gatica"
						},
						"bug_when": "2024-04-18 20:35:30 +0000",
						"thetext": "\nMore information about this security flaw is available in the following bug:\n\nhttp://bugzilla.redhat.com/show_bug.cgi?id=2275989\n\nDisclaimer: Community trackers are created by Red Hat Product Security team on a best effort basis. Package maintainers are required to ascertain if the flaw indeed affects their package, before starting the update process."
					},
					{
						"isprivate": "0",
						"commentid": "17978092",
						"comment_count": "1",
						"who": {
							"text": "rgatica",
							"name": "Robb Gatica"
						},
						"bug_when": "2024-04-18 20:35:32 +0000",
						"thetext": "Use the following template to for the 'fedpkg update' request to submit an\nupdate for this issue as it contains the top-level parent bug(s) as well as\nthis tracking bug.  This will ensure that all associated bugs get updated\nwhen new packages are pushed to stable.\n\n=====\n\n# bugfix, security, enhancement, newpackage (required)\ntype=security\n\n# low, medium, high, urgent (required)\nseverity=medium\n\n# testing, stable\nrequest=testing\n\n# Bug numbers: 1234,9876\nbugs=2275989,2275991\n\n# Description of your update\nnotes=Security fix for [PUT CVEs HERE]\n\n# Enable request automation based on the stable/unstable karma thresholds\nautokarma=True\nstable_karma=3\nunstable_karma=-3\n\n# Automatically close bugs when this marked as stable\nclose_bugs=True\n\n# Suggest that users restart after update\nsuggest_reboot=False\n\n======\n\nAdditionally, you may opt to use the bodhi web interface to submit updates:\n\nhttps://bodhi.fedoraproject.org/updates/new"
					},
					{
						"isprivate": "0",
						"commentid": "17978106",
						"comment_count": "2",
						"who": {
							"text": "code",
							"name": "Ben Beasley"
						},
						"bug_when": "2024-04-18 20:51:08 +0000",
						"thetext": "Updates to 3.9.5 for all releases should be possible. Ideally, the security updates with llhttp 9.1.2 currently in testing will reach stable first, since aiohttp 3.9.5 will require the updated llhttp.\n\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-2f15e6e876\n\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-f83b123d63\n\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-5dc487ee89"
					},
					{
						"isprivate": "0",
						"commentid": "17982188",
						"comment_count": "3",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-04-22 13:56:19 +0000",
						"thetext": "FEDORA-2024-c4a71dab58 (python-aiohttp-3.9.5-1.fc41) has been submitted as an update to Fedora 41.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-c4a71dab58"
					},
					{
						"isprivate": "0",
						"commentid": "17982204",
						"comment_count": "4",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-04-22 14:00:14 +0000",
						"thetext": "FEDORA-2024-c4a71dab58 (python-aiohttp-3.9.5-1.fc41) has been pushed to the Fedora 41 stable repository.\nIf problem still persists, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "17984298",
						"comment_count": "5",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-04-23 16:37:29 +0000",
						"thetext": "FEDORA-2024-000a25f3fc (python-aiohttp-3.9.5-1.fc40 and python-openapi-core-0.19.1-3.fc40) has been submitted as an update to Fedora 40.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-000a25f3fc"
					},
					{
						"isprivate": "0",
						"commentid": "17984689",
						"comment_count": "6",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-04-23 18:04:33 +0000",
						"thetext": "FEDORA-2024-e0057e6044 (python-aiohttp-3.9.5-1.fc39) has been submitted as an update to Fedora 39.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-e0057e6044"
					},
					{
						"isprivate": "0",
						"commentid": "17984731",
						"comment_count": "7",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-04-23 18:34:45 +0000",
						"thetext": "FEDORA-2024-f34786d26f (python-aiohttp-3.9.5-1.fc38) has been submitted as an update to Fedora 38.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-f34786d26f"
					},
					{
						"isprivate": "0",
						"commentid": "17985379",
						"comment_count": "8",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-04-24 01:49:26 +0000",
						"thetext": "FEDORA-2024-000a25f3fc has been pushed to the Fedora 40 testing repository.\nSoon you'll be able to install the update with the following command:\n`sudo dnf upgrade --enablerepo=updates-testing --refresh --advisory=FEDORA-2024-000a25f3fc`\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2024-000a25f3fc\n\nSee also https://fedoraproject.org/wiki/QA:Updates_Testing for more information on how to test updates."
					},
					{
						"isprivate": "0",
						"commentid": "17985408",
						"comment_count": "9",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-04-24 02:01:33 +0000",
						"thetext": "FEDORA-2024-e0057e6044 has been pushed to the Fedora 39 testing repository.\nSoon you'll be able to install the update with the following command:\n`sudo dnf upgrade --enablerepo=updates-testing --refresh --advisory=FEDORA-2024-e0057e6044`\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2024-e0057e6044\n\nSee also https://fedoraproject.org/wiki/QA:Updates_Testing for more information on how to test updates."
					},
					{
						"isprivate": "0",
						"commentid": "17985478",
						"comment_count": "10",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-04-24 03:00:19 +0000",
						"thetext": "FEDORA-2024-f34786d26f has been pushed to the Fedora 38 testing repository.\nSoon you'll be able to install the update with the following command:\n`sudo dnf upgrade --enablerepo=updates-testing --refresh --advisory=FEDORA-2024-f34786d26f`\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2024-f34786d26f\n\nSee also https://fedoraproject.org/wiki/QA:Updates_Testing for more information on how to test updates."
					},
					{
						"isprivate": "0",
						"commentid": "17998030",
						"comment_count": "11",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-05-02 01:37:22 +0000",
						"thetext": "FEDORA-2024-e0057e6044 (python-aiohttp-3.9.5-1.fc39) has been pushed to the Fedora 39 stable repository.\nIf problem still persists, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "17998048",
						"comment_count": "12",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-05-02 01:43:36 +0000",
						"thetext": "FEDORA-2024-f34786d26f (python-aiohttp-3.9.5-1.fc38) has been pushed to the Fedora 38 stable repository.\nIf problem still persists, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "17998070",
						"comment_count": "13",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-05-02 01:57:00 +0000",
						"thetext": "FEDORA-2024-000a25f3fc (python-aiohttp-3.9.5-1.fc40 and python-openapi-core-0.19.1-3.fc40) has been pushed to the Fedora 40 stable repository.\nIf problem still persists, please make note of it in this bug report."
					}
				]
			}
		}
	],
	"builds": [
		{
			"epoch": 0,
			"nvr": "python-aiohttp-3.9.5-1.fc41",
			"release_id": 80,
			"signed": true,
			"type": "rpm",
			"package": {
				"aarch64": [
					{
						"name": "python-aiohttp-debugsource",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "aarch64"
					},
					{
						"name": "python3-aiohttp-debuginfo",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "aarch64"
					},
					{
						"name": "python3-aiohttp",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "aarch64"
					},
					{
						"name": "python3-aiohttp+speedups",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "aarch64"
					}
				],
				"i686": [
					{
						"name": "python3-aiohttp-debuginfo",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "i686"
					},
					{
						"name": "python3-aiohttp",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "i686"
					},
					{
						"name": "python-aiohttp-debugsource",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "i686"
					},
					{
						"name": "python3-aiohttp+speedups",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "i686"
					}
				],
				"ppc64le": [
					{
						"name": "python3-aiohttp",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "ppc64le"
					},
					{
						"name": "python-aiohttp-debugsource",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "ppc64le"
					},
					{
						"name": "python3-aiohttp-debuginfo",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "ppc64le"
					},
					{
						"name": "python3-aiohttp+speedups",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "ppc64le"
					}
				],
				"s390x": [
					{
						"name": "python3-aiohttp",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "s390x"
					},
					{
						"name": "python-aiohttp-debugsource",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "s390x"
					},
					{
						"name": "python3-aiohttp-debuginfo",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "s390x"
					},
					{
						"name": "python3-aiohttp+speedups",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "s390x"
					}
				],
				"src": [
					{
						"name": "python-aiohttp",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "src"
					}
				],
				"x86_64": [
					{
						"name": "python3-aiohttp-debuginfo",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "x86_64"
					},
					{
						"name": "python-aiohttp-debugsource",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "x86_64"
					},
					{
						"name": "python3-aiohttp+speedups",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "x86_64"
					},
					{
						"name": "python3-aiohttp",
						"version": "3.9.5",
						"release": "1.fc41",
						"arch": "x86_64"
					}
				]
			}
		}
	],
	"close_bugs": true,
	"comments": [
		{
			"id": 3500031,
			"text": "This update was automatically created",
			"timestamp": "2024-04-22 13:56:16",
			"update_id": 606388,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3500032,
			"text": "This update's test gating status has been changed to 'ignored'.",
			"timestamp": "2024-04-22 13:56:27",
			"update_id": 606388,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3500043,
			"text": "This update's test gating status has been changed to 'waiting'.",
			"timestamp": "2024-04-22 13:58:24",
			"update_id": 606388,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3500044,
			"text": "This update's test gating status has been changed to 'ignored'.",
			"timestamp": "2024-04-22 13:58:24",
			"update_id": 606388,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3500047,
			"text": "This update has been submitted for stable by bodhi",
			"timestamp": "2024-04-22 14:00:11",
			"update_id": 606388,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		}
	],
	"content_type": "rpm",
	"critpath_groups": "",
	"date_approved": "2024-04-22 14:00:11",
	"date_pushed": "2024-04-22 14:00:11",
	"date_stable": "2024-04-22 14:00:11",
	"date_submitted": "2024-04-22 13:56:16",
	"date_testing": "2024-04-22 13:58:24",
	"meets_testing_requirements": true,
	"notes": "Automatic update for python-aiohttp-3.9.5-1.fc41.\n\n##### **Changelog**\n\n```\n* Fri Apr 19 2024 Benjamin A. Beasley <code@musicinmybrain.net> - 3.9.5-1\n- Update to 3.9.5 (fix RHBZ#2275991, fix CVE-2024-27306)\n\n```",
	"pushed": true,
	"release": {
		"branch": "rawhide",
		"candidate_tag": "f41-updates-candidate",
		"create_automatic_updates": true,
		"dist_tag": "f41",
		"id_prefix": "FEDORA",
		"long_name": "Fedora 41",
		"mail_template": "fedora_errata_template",
		"name": "F41",
		"override_tag": "f41-override",
		"package_manager": "unspecified",
		"pending_signing_tag": "f41-signing-pending",
		"pending_stable_tag": "f41-updates-pending",
		"pending_testing_tag": "f41-updates-testing-pending",
		"stable_tag": "f41",
		"state": "pending",
		"testing_tag": "f41-updates-testing",
		"version": "41"
	},
	"severity": "unspecified",
	"stable_karma": 3,
	"status": "stable",
	"suggest": "unspecified",
	"test_gating_status": "ignored",
	"title": "python-aiohttp-3.9.5-1.fc41",
	"type": "security",
	"url": "https://bodhi.fedoraproject.org/updates/FEDORA-2024-c4a71dab58",
	"unstable_karma": -3,
	"updateid": "FEDORA-2024-c4a71dab58",
	"user": {
		"avatar": "https://seccdn.libravatar.org/avatar/30796d789ab4befafdec48a7b896ed471c9c78917b2e98852ffb42d4c9b7661c?s=24&d=retro",
		"email": "code@musicinmybrain.net",
		"groups": [
			{
				"name": "packager"
			},
			{
				"name": "python-sig"
			},
			{
				"name": "ipausers"
			},
			{
				"name": "fedora-contributor"
			},
			{
				"name": "signed_fpca"
			},
			{
				"name": "fedorabugs"
			},
			{
				"name": "neuro-sig"
			},
			{
				"name": "fesco"
			},
			{
				"name": "python-packagers-sig"
			}
		],
		"id": 5936,
		"name": "music",
		"openid": "music.id.fedoraproject.org"
	},
	"version_hash": "99faa1a817767957b29a79d08593a361f2ae11c8"
}
